src/fidesops/schemas/email/email.py

```diff
@@ -0,0 +1,155 @@
+from enum import Enum
+from typing import Union, List, Any, Dict, Optional
+
+from pydantic import BaseModel, Extra, validator
+
+from fidesops.common_exceptions import ValidationError
+from fidesops.schemas import Msg
+from fidesops.schemas.api import BulkResponse, BulkUpdateFailed
+from fidesops.schemas.shared_schemas import FidesOpsKey
+
+
+class EmailServiceType(Enum):
+    """Enum for email service type"""
+    # may support twilio or google in the future
+    MAILGUN = "mailgun"
+
+
+class EmailPurpose(Enum):  # support using different email services for different purposes
+    """Enum for email usage type"""
+    # verify email upon acct creation
+    SUBJECT_IDENTITY_VERIFICATION = "subject_identity_verification"
+    # e.g. your request is processing
+    AUTOMATED_SUBJECT_COMMUNICATION = "automated_subject_communication"
```

Comment:
We could be adding more complexity than we need here. If all emails that can be sent from Fidesops are transactional, do we need to differentiate between these use cases? I don't _think_ it's in scope to enable setups where Fidesops can send mail from one connector for subject ID verification, and another for maildrop for instance.  @adriaaaa can you confirm?

I can imagine that as a very interesting increment down the line however — to say something like "I want mail to this SaaS to come from this domain"

Reply:
I'm also removing these enums, since we're not going to support using different email providers for different purposes.